### ISSUE TITLE
Jupyter lab build fails in fresh conda environment

### ISSUE THREAD

**Issue by Losoly** [jupyterlab/jupyterlab#7158]
## Description
Since I updated my Jupiterlab Enviroment to 1.0.2 I have a problem with my Jupyter Lab enviroment. The icons on the navigation bar are missing. 

This error pops up intermittenly where the icons should be: 
This page contains the following errors: error on line 1 at column 1: Document is empty

![image](https://user-images.githubusercontent.com/51330045/64428647-912f5600-d0b4-11e9-9c60-a670fd6418ca.png)

I cannot reproduce this issue in a clean environment. I already removed all my Jupter Lab extensions, did jupyter lab clean and jupyter lab build but still the same.

## Context
Name                    Version                   Build  Channel
jupyter                   1.0.0                    py37_7
jupyter-contrib-core      0.3.3                    pypi_0    pypi
jupyter-contrib-nbextensions 0.5.1                    pypi_0    pypi
jupyter-highlight-selected-word 0.2.0                    pypi_0    pypi
jupyter-latex-envs        1.4.6                    pypi_0    pypi
jupyter-nbextensions-configurator 0.4.1                    pypi_0    pypi
jupyter_client            5.3.1                      py_0
jupyter_console           6.0.0                    py37_0
jupyter_core              4.5.0                      py_0
jupyterlab                1.0.2            py37hf63ae98_0
jupyterlab-github         1.0.0                    pypi_0    pypi
jupyterlab_server         1.0.0                      py_1
jupyterthemes             0.20.0                   pypi_0    pypi

**Comment by bmwilly** [comment]
I'm having the same problem. AFAICT, it only seems to occur if I add extensions or run `jupyter lab build`.

**Comment by icks** [comment]
I just encountered the exact same error. I had some remaining JupyterLab directories after purging my Anaconda folder and just noticed that I still had an older JupyterLab version after the new Anaconda installation:

```
$ jupyter --version | grep lab
jupyter lab      : 1.0.2
```

Upgrading through `pip` to the newest version and rebuilding the assets worked for me:

```
$ pip install -U jupyterlab
[...]
$ jupyter --version | grep lab
jupyter lab      : 1.1.1
$ jupyter lab build
[...]
```

**Comment by blink1073** [comment]
Hi folks, we haven't yet tracked down the root cause but it appears to be a backwards incompatibility in one of the JavaScript packages in our 1.1 series.  Once we identify the problem we will release a new set of 1.1 JS package(s) that are backward compatible.  In the mean time, please use upgrade to the 1.1 series if possible.  `conda install -c conda-forge jupyterlab` or `pip install --upgrade jupyterlab`.

**Comment by blink1073** [comment]
@telamonian, I think this one might be related to #6034 as well:

![image](https://user-images.githubusercontent.com/2096628/64474328-5ae7ea00-d139-11e9-895c-67416d104978.png)

**Comment by blink1073** [comment]
To reproduce the above:  

```
pip install jupyterlab==1.0.2
jupyter lab build
jupyter lab
```

**Comment by Losoly** [comment]
> I just encountered the exact same error. I had some remaining JupyterLab directories after purging my Anaconda folder and just noticed that I still had an older JupyterLab version after the new Anaconda installation:
> 
> ```
> $ jupyter --version | grep lab
> jupyter lab      : 1.0.2
> ```
> 
> Upgrading through `pip` to the newest version and rebuilding the assets worked for me:
> 
> ```
> $ pip install -U jupyterlab
> [...]
> $ jupyter --version | grep lab
> jupyter lab      : 1.1.1
> $ jupyter lab build
> [...]
> ```

This was the solution for my issue. Thank you very much! 😀

**Comment by jasongrout** [comment]
Original problem fixed by #7182
